assets/pages/new-contributions-landing/components/StripePaymentRequestButton.jsx

```diff
@@ -92,6 +93,7 @@ function updateUserEmail(data: Object, setEmail: string => void) {
 // Calling the complete function will close the pop up payment window
 const onComplete = (complete: Function) => (res: PaymentResult) => {
   if (res.paymentStatus === 'success') {
+    sendClickedEvent('apple-pay-payment-complete');
     complete('success');
   } else if (res.paymentStatus === 'failure') {
```

Comment:
separate to this pr, but do we need to track failure somewhere (e.g. sentry)

Reply:
it is tracked in `onPaymentResult` in `contributionsLandingActions` already